CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog].
 
+## 3.3 (released 2019-07-10)
```

Comment:
Label this Unreleased. I don't make a release until the changes have been master for several weeks -- otherwise there wouldn't be any stability guarantee.

Reply:
Sure, I follow you.

Reply:
Which is good for you? `## 3.2 (unreleased)` or `## 3.2 (work in progress)`.

Reply:
Look at the commit history: https://github.com/raxod502/prescient.el/commit/89584b8fe8ac2557c152298eb87b6e7662237bab

Reply:
Thank you for your kind instruction. I fixed it.